src/components/reactions.vue

```diff
@@ -102,5 +102,26 @@ export default defineComponent({
 })
 </script>
 <style scoped>
+.plusButton{
+  -webkit-font-smoothing: antialiased;
+  margin: 4px;
+  padding: 7px 1px;
+  display: inline-flex;
+  position: relative;
+  align-items: center;
+  height: 32px;
+  background: var(--background);
+  color: var(--color);
+  font-family: var(--ion-font-family, inherit);
+  font-size: 14px;
+  line-height: 1;
+  cursor: pointer;
+  overflow: hidden;
+  vertical-align: middle;
+}
+
+.plusButton:hover, .plusButton:focus {
+  border: 1px solid #c1c1c1;
+}
```

Comment:
This is `ion-button`, it shouldn't need these extra states. Rather than adding them, let's figure out why they aren't applied.

And as said before, let's refrain from adding colors and specific px and alike but reuse `var(--ion-border)` and other variables, so theming stays possible.